CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ and this project adheres to
 - `MemoryWritable` internal buffer size is now limited to 8 MB by default.
 - Signature of `callerFilepath()` to allow passing `RegExp` as depth to be used
   for filtering of stack frames.
+- `cryptoPrefetcher()` to implement Iterator protocol.
```

Comment:
This is technically incorrect and misleading since now it only implements [the *Iterable* interface](https://tc39.es/ecma262/#sec-iterable-interface), not the Iterator protocol.

Besides that, I think that it's better to rewrite it as follows:
```suggestion
- Return value of `cryptoPrefetcher()` now implements the Iterable interface.
```